sdk/search/search-documents/src/searchClient.ts

```diff
@@ -931,16 +929,7 @@ export class SearchClient<TModel extends object> implements IndexDocumentsClient
     return output;
   }
 
-  private convertVectorQuery(): undefined;
-  private convertVectorQuery(vectorQuery: VectorizedQuery<TModel>): GeneratedVectorizedQuery;
-  private convertVectorQuery(
-    vectorQuery: VectorizableTextQuery<TModel>,
-  ): GeneratedVectorizableTextQuery;
-  private convertVectorQuery(vectorQuery: VectorQuery<TModel>): GeneratedVectorQuery;
-  private convertVectorQuery(vectorQuery?: VectorQuery<TModel>): GeneratedVectorQuery | undefined {
-    if (!vectorQuery) {
-      return vectorQuery;
-    }
+  private convertVectorQuery<T extends VectorQuery<TModel>>(vectorQuery: T): GeneratedVectorQuery {
```

Comment:
TODO: toGenerated / fromGenerated naming conventions + this can be a function. Not blocking

Reply:
Sounds good. I'll open an issue tracking migrating all such functions to their own file/namespace as well.